common/src/main/java/io/pravega/common/concurrent/FutureHelpers.java

```diff
@@ -109,6 +109,19 @@
     public static <T> boolean isSuccessful(CompletableFuture<T> f) {
         return f.isDone() && !f.isCompletedExceptionally() && !f.isCancelled();
     }
+    
+    /**
+     * If the future has failed returns the exception that caused it. Otherwise returns null.
+     * @return null or the exception that caused the Future to fail.
```

Comment:
Please fix this javadoc parameters. 
add these  after the first line and before @ return.
 +      *
 +     * @param future   The future to inspect.
 +     * @param <T> The Type of the future's result.